js/runner.go

```diff
@@ -363,57 +362,46 @@ type VU struct {
 	TLSConfig *tls.Config
 	ID        int64
 	Iteration int64
+	runMutex  *sync.Mutex
```

Comment:
This can be a part of the `ActiveVU` - after all you can't run the `VU` anymore.

Reply:
So moving this to `ActiveVU` actually causes `TestVURunInterruptDoesntPanic` to panic at https://github.com/loadimpact/k6/blob/2c8a111d37cba2780f7c77658bc03c5940adfad9/js/runner.go#L407

See https://circleci.com/gh/loadimpact/k6/9726 .

So I guess it needs to be defined on `js.VU` after all?